src/main/frontend/components/whiteboard.cljs

```diff
@@ -188,6 +188,9 @@
     (fn [e]
       (util/stop e)
       (whiteboard-handler/create-new-whiteboard-and-redirect!))}
+   [:div
```

Comment:
Please remove as we don't want new UI in a translation PR. You can create a separate issue or PR if this is something you're still interested

Reply:
@logseq-cldwalker This is a hidden element to use the translation tag, otherwise it will report this translation is never used in UI. I used it ":whiteboard/create-or-search-graph" in tsx file, not directly in cljs file.